aptos-move/framework/aptos-framework/sources/resource_account.spec.move

```diff
@@ -60,6 +74,27 @@ spec aptos_framework::resource_account {
         aborts_if !get && !(exists<Account>(resource_addr) && len(optional_auth_key) == 32);
     }
 
+    spec schema RotateAccountAuthenticationKeyAndStoreCapabilityAbortsIfWithoutAccountLimit {
```

Comment:
I am wondering if we can include RotateAccountAuthenticationKeyAndStoreCapabilityAbortsIf to reduce duplicate?

Reply:
well as you may see, there is a little difference between these two schema, which are the last two lines: